Std/Lean/Elab/Command.lean

```diff
@@ -0,0 +1,53 @@
+/-
+Copyright (c) 2023 Mac Malone. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Mac Malone
+-/
+import Lean.Elab.Command
+
+/-!
+# Command elaboration helpers
+-/
+
+namespace Lean.Elab.Command
+
+/--
+For each named option which exists, set its value.
+Will still fail if the set value does not match the option's proper type
+(e.g., setting a string value to a boolean option).
+-/
+def trySetOptions (settings : Array (Name × DataValue)) : CommandElabM PUnit := do
```

Comment:
How about also introduce `def trySetOption (name : Name) (val : DataValue) : CommandElabM PUnit` and define this in terms of that?  It looks like there wouldn't be a significant performance cost.

Reply:
@joehendrix I added `trySetOption`, but it did not seem worth it to define `trySetOptions` in terms of it  (the number of references loads would be `O(n)` instead of `O(1)`).